doc/api/tty.md

```diff
@@ -2,13 +2,19 @@
 
     Stability: 2 - Stable
 
-The `tty` module houses the `tty.ReadStream` and `tty.WriteStream` classes. In
-most cases, you will not need to use this module directly.
+The `tty` modules provides the `tty.ReadStream` and `tty.WriteStream` classes.
+In most cases, it will not be necessary to use this module directly. However,
```

Comment:
`necessary or possible`

This is still misleading though, these apis will be available for use on stdio, but you will not be able to create the streams manually without access to either:
- A native module to fake a TTY
- An external TTY without running the process in a TTY (somehow?)